catalogs/catalog-jdbc-doris/src/main/java/com/datastrato/gravitino/catalog/doris/operation/DorisDatabaseOperations.java

```diff
@@ -4,28 +4,121 @@
  */
 package com.datastrato.gravitino.catalog.doris.operation;
 
+import com.datastrato.gravitino.catalog.doris.utils.DorisUtils;
 import com.datastrato.gravitino.catalog.jdbc.JdbcSchema;
 import com.datastrato.gravitino.catalog.jdbc.operation.JdbcDatabaseOperations;
 import com.datastrato.gravitino.exceptions.NoSuchSchemaException;
+import com.datastrato.gravitino.exceptions.NoSuchTableException;
+import com.datastrato.gravitino.meta.AuditInfo;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Collections;
 import java.util.Map;
+import org.apache.commons.lang3.StringUtils;
 
 /** Database operations for Doris. */
 public class DorisDatabaseOperations extends JdbcDatabaseOperations {
-  // TODO: add implementation for doris catalog
 
   @Override
-  public JdbcSchema load(String databaseName) throws NoSuchSchemaException {
-    throw new UnsupportedOperationException();
+  public String generateCreateDatabaseSql(
+      String databaseName, String comment, Map<String, String> properties) {
+    if (StringUtils.isNotEmpty(comment)) {
+      throw new UnsupportedOperationException(
+          "Doris doesn't support set database comment: " + comment);
+    }
+    StringBuilder sqlBuilder = new StringBuilder();
+
+    // Append database name
+    sqlBuilder.append(String.format("CREATE DATABASE `%s`", databaseName));
+
+    // Append properties
+    sqlBuilder.append(DorisUtils.generatePropertiesSql(properties));
+
+    String result = sqlBuilder.toString();
+    LOG.info("Generated create database:{} sql: {}", databaseName, result);
+    return result;
   }
 
   @Override
-  protected String generateCreateDatabaseSql(
-      String databaseName, String comment, Map<String, String> properties) {
-    throw new UnsupportedOperationException();
+  public String generateDropDatabaseSql(String databaseName, boolean cascade) {
+    StringBuilder sqlBuilder = new StringBuilder();
+    sqlBuilder.append(String.format("DROP DATABASE `%s`", databaseName));
+    if (cascade) {
+      sqlBuilder.append(" FORCE");
+      return sqlBuilder.toString();
+    }
+
+    try (final Connection connection = this.dataSource.getConnection()) {
```

Comment:
Can we only use one try statement here? It seems excessive.

Reply:
done.

Reply:
done